app/experimenter/kinto/tasks.py

```diff
@@ -397,3 +400,29 @@ def nimbus_synchronize_preview_experiments_in_kinto():
         metrics.incr("nimbus_synchronize_preview_experiments_in_kinto.failed")
         logger.info(f"Synchronizing preview experiments failed: {e}")
         raise e
+
+
+@app.task
+@metrics.timer_decorator("send_end_enrollment_email")
+def nimbus_send_end_enrollment_email():
+    """
+    A scheduled task that checks for any experiments that
+    should have their enrollment turned off and fires off
+    a reminder email
+    """
+    metrics.incr("send_end_enrollment_email.started")
+
+    experiments = NimbusExperiment.objects.filter(
+        status=NimbusExperiment.Status.LIVE,
+    )
+
+    for experiment in experiments:
+        if (
+            experiment.should_end_enrollment
+            and not experiment.emails.filter(
+                type=NimbusExperiment.EmailType.ENROLLMENT_END
+            ).exists()
+        ):
+            nimbus_send_enrollment_ending_email(experiment)
+
+    metrics.incr("send_end_enrollment_email.completed")
```

Comment:
Also yeah, you're right we need a task for this.  Actually what would be really nice is also moving the end experiment email check out of the other task into this one and just calling the 'send_reminder_emails_task' or some such, so we've got them all in one place and the other tasks can just focus on checking statuses.  Filed a followup: https://github.com/mozilla/experimenter/issues/6001